rocket_profile.py

```diff
@@ -0,0 +1,20 @@
+from typing import Dict, List
+
+from label import Label, default_labels
+
+
+class RocketProfile:
+    def __init__(self, buttons: Dict[str, str], labels: List[Label]):
+        self.buttons = buttons
+        self.labels = labels
+
+    def update_labels(self, main_window) -> None:
+        for label in self.labels:
+            exec(f"main_window.{label.name}Label.setText(label.update(main_window))")
```

Comment:
```suggestion
            getattr(main_window, label.name + "Label").setText(label.update(main_window))
```

Would something like this work? This might help get rid of the exec, make it more debuggable and understandable by the IDE?

Reply:
I'll try that out. Thanks!